app/hotels/src/singleHotel/hotelInformation/location/Location.js

```diff
@@ -68,31 +68,24 @@ export class Location extends React.Component<PropsWithContext> {
                 <Translation passThrough={idx(address, _ => _.city)} />
               </Text>
             </View>
-            <View style={styles.rightColumn}>
-              {typeof latitude === 'number' &&
-                typeof longitude === 'number' && (
-                  <MapView
-                    region={{
-                      latitude: latitude,
-                      longitude: longitude - 0.005, // move center little bit right
-                      latitudeDelta: 0.01,
-                      longitudeDelta: 0.01,
-                    }}
-                    scrollEnabled={false}
-                    style={[StyleSheet.absoluteFillObject, styles.mapBottom]}
-                  >
-                    <MapView.Marker
-                      coordinate={{
-                        latitude,
-                        longitude,
-                      }}
-                    >
-                      <DropMarker size={30} />
-                    </MapView.Marker>
-                  </MapView>
-                )}
-              <StretchedImage source={gradient} />
+            {typeof latitude === 'number' &&
+              typeof longitude === 'number' && (
+                <MapView
+                  region={{
+                    latitude: latitude + 0.001, // move center little bit bottom
+                    longitude: longitude - 0.025, // move center little bit right
+                    latitudeDelta: 0.01,
+                    longitudeDelta: 0.01,
+                  }}
+                  scrollEnabled={false}
+                  style={[StyleSheet.absoluteFillObject, styles.mapBottom]}
+                />
+              )}
+
+            <View style={styles.overlayMarker}>
```

Comment:
This is a bit of a hack, but I aligned the icon from the previously rendered code with this one. This allows us to use zIndex.